From ed3708642a0a32b87267a13506509da47bbc67aa Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Sun, 10 Apr 2016 14:14:10 -0700
Subject: [PATCH 04/31] Add error message string for imap_mailbox_rename() on
 root folder.

This was fixed just before 1.6.0, but due to the string freeze, could
not add an error message.
---
 imap/browse.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/imap/browse.c b/imap/browse.c
index ccbf1c6..04d0c0b 100644
--- a/imap/browse.c
+++ b/imap/browse.c
@@ -319,11 +319,9 @@ int imap_mailbox_rename(const char* mailbox)
     goto fail;
   }
 
-  /* TODO: add mutt_error call, such as
-   * "Cannot rename root folder"
-   */
   if (!mx.mbox)
   {
+    mutt_error _("Cannot rename root folder");
     goto fail;
   }
 
-- 
2.8.2

